packages/SystemUI/src/com/android/systemui/biometrics/FODCircleView.java

```diff
@@ -151,7 +151,7 @@
         R.drawable.fod_icon_pressed_cyan,
         R.drawable.fod_icon_pressed_green,
         R.drawable.fod_icon_pressed_yellow,
-        R.drawable.fod_icon_pressed_ffffe6
+        R.drawable.fod_icon_pressed_light_yellow
```

Comment:
why modify FOD stuff?

Reply:
I think that's from the force push of @neobuddy89